images/build/cross/variants.yaml

```diff
@@ -1,8 +1,8 @@
 variants:
   canary:
     CONFIG: 'canary'
-    GO_VERSION: '1.16rc1'
-    IMAGE_VERSION: 'v1.16.0-rc.1-canary-1'
```

Comment:
now looking at it again, i think this should stay as 1.16 , will revert this part